android/src/main/java/com/hoxfon/react/RNTwilioVoice/TwilioVoiceModule.java

```diff
@@ -492,6 +494,14 @@ public void initWithAccessToken(final String accessToken, Promise promise) {
             return;
         }
 
+        if(!checkPermissionForMicrophone()){
```

Comment:
formatting, please be consistent with the rest of java code. Maybe use Android Studio to save the file, it should then format automatically. Sorry I didn't figure out how to add a linter yet.

Reply:
I think there's a prettier for java , we can implement it and adding a pre push task.